probe-dictionary/explore.js

```diff
@@ -967,7 +967,9 @@ function getMeasurementCountsPerVersion() {
           // If so, increase the count.
           if (recording) {
             let k = recording.optout ? "optout" : "optin";
-            data[k] += 1;
+            if (k === "optout") {
```

Comment:
I think this means we will only count "optout" probes, no matter what channel is selected.
Have you tried selecting the nightly or beta channel and see if they still show both optin and optout probe counts?

The behavior we want is:
- When the release channel is selected, count only "optout" probes.
- When any other channel is selected, count both "optout" and "optin" probes.